smart_canvas/render.py

```diff
@@ -35,6 +39,10 @@ def render(self, _time, frame_time):
     def close(self):
         self.video.stop()
         self.core.stop()
+    
+    def resized(self, width, height):
+        vp = self.ctx.fbo.viewport
+        self.win_size = (vp[2] - vp[0], vp[3] - vp[1])
```

Comment:
When do we call this?
I see that you have the screensize/win_size here and in the core.py.

Reply:
Callback from Window class.

Reply:
Oh, so when window resize happens, then this gets called, right?